tests/README.md

```diff
@@ -2,33 +2,38 @@
 
 This is the comdb2 test suite.
 
+### Prerequisites
+
 Some of the tests require non standard tools to be installed, so here is a 
 list of packages needed to run all the tests:
 
 `apt-get install bash coreutils jq tcl-dev`
 
-Clustered tests also need ssh-client and ssh-server setup.
```

Comment:
I don't think ssh-client and ssh-server here refer to specific software packages. I'd rephrase the sentence to "Cluster tests also require SSH."